custom_components/drivvo/sensors.py

```diff
@@ -76,7 +75,6 @@ class DrivvoSensorEntityDescription(SensorEntityDescription):
         translation_key="refuelling_general_average",
         name="Refuelling General Average",
         icon="mdi:fuel",
-        device_class=SensorDeviceClass.VOLUME_FLOW_RATE,
```

Comment:
Fixes:
```
2025-03-03 00:42:33.053 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.xxxxxx_media_geral_de_combustivel (<class 'custom_components.drivvo.sensor.DrivvoSensorEntity'>) is using native unit of measurement 'km/L' which is not a valid unit for the device class ('volume_flow_rate') it is using; expected one of ['gal/min', 'mL/s', 'L/min', 'm³/h', 'ft³/min']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/hudsonbrendon/sensor.drivvo/issues
```